core/templates/dev/head/components/ResponseHeaderDirective.js

```diff
@@ -48,6 +47,11 @@ oppia.directive('responseHeader', [
             return outcome && (outcome.dest === activeStateName);
           };
 
+          $scope.isCorrectnessFeedbackEnabled = function() {
+            return (
+              explorationCorrectnessFeedbackService.isEnabled());
```

Comment:
I guess this can now go on the previous line (and the surrounding parens removed)?

Reply:
Done!